src/snowflake/connector/ocsp_snowflake.py

```diff
@@ -59,39 +56,57 @@
 )
 from snowflake.connector.errors import RevocationCheckError
 from snowflake.connector.network import PYTHON_CONNECTOR_USER_AGENT
-from snowflake.connector.ssd_internal_keys import (
-    ocsp_internal_dep1_key_ver,
-    ocsp_internal_dep2_key_ver,
-    ocsp_internal_ssd_pub_dep1,
-    ocsp_internal_ssd_pub_dep2,
-)
 from snowflake.connector.telemetry_oob import TelemetryService
 from snowflake.connector.time_util import DecorrelateJitterBackoff
 
 from . import constants
 from .cache import SFDictCache, SFDictFileCache
 from .telemetry import TelemetryField, generate_telemetry_data_dict
 
+
+class OCSPResponseValidationMetadata(NamedTuple):
+    exception: Exception | None = None
+    issuer: Certificate | None = None
+    subject: Certificate | None = None
+    cert_id: CertId | None = None
+    ocsp_response: bytes | None = None
+
+
+class OCSPResponseValidationResult(NamedTuple):
+    ocsp_response_validation_metadata: OCSPResponseValidationMetadata | None = None
```

Comment:
Do you think that we could flatten the 2 NamedTuples into just one?

Reply:
yes, updated to just use one